integration-test/tests/email-autofill.extension.spec.js

```diff
@@ -66,4 +66,32 @@ test.describe('chrome extension', () => {
             'autofill_show', 'autofill_private_address' // second private autofill
         ])
     })
+
+    test('should not close the modal when autofilling', async ({page}) => {
```

Comment:
Existing tests should have caught the regression already, but I'm adding this one specific to the extensions because when we remove all the Catalina stuff the bug won't actually happen anymore in the desktop apps, so this gives us more confidence going forward.

Side note, it will be quite an undertaking to rewrite all these integration tests when we remove the in-page Catalina pulldown 😰. Almost all macOS integration tests rely on the Catalina version of the pulldown.